src/components/molecules/PlayaTime/PlayaTime.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import dayjs from "dayjs";
+import timezone from "dayjs/plugin/timezone";
+import updateLocale from "dayjs/plugin/updateLocale";
+import utc from "dayjs/plugin/utc";
+
+import {
+  CURRENT_TIME_IN_LOCATION,
+  CURRENT_TIMEZONE,
+  PLAYA_TIME_REFRESH_MS,
+} from "settings";
+
+import { useInterval } from "hooks/useInterval";
+
+import "./PlayaTime.scss";
+
+// @debt deprecated dayjs library, use date-fns instead
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.extend(updateLocale);
+
+export const PlayaTime: React.FC = () => {
+  const [currentTime, setCurrentTime] = useState("");
+  const setPlayaTime = () => {
+    const pt = dayjs().tz(CURRENT_TIMEZONE);
+    setCurrentTime(pt.format("h:mm A"));
+  };
+
+  useEffect(setPlayaTime, []);
+
+  useInterval(() => {
+    setPlayaTime();
+  }, PLAYA_TIME_REFRESH_MS);
```

Comment:
can we do it this way?
```suggestion
  useInterval(setPlayaTime, PLAYA_TIME_REFRESH_MS);
```